zetaclient/context/app_test.go

```diff
@@ -69,37 +70,48 @@ func TestNew(t *testing.T) {
 	})
 
 	t.Run("should create new zetacore context with config containing evm chain params", func(t *testing.T) {
-		testCfg := config.New(false)
+		// ARRANGE
```

Comment:
There are some remaining `zetacore context`, `coreContext` in the codebase. Since we removed the concept of zetacore context, leaving only app context, we need a text search and replace them with `app context` and `appContext`.

Reply:
Created an issue https://github.com/zeta-chain/node/issues/2526